lib/stats.rb

```diff
@@ -178,7 +178,18 @@ def teams_hash
   end
 
   def team_info
-
+    team_info_hash = Hash.new { |hash, key| hash[key] = [] }
+
+    @teams_data.each do |team|
+      team_info_hash[team[:team_id]] ={
+        team_id: team[:team_id],
+        franchise_id: team[:franchiseid],
+        team_name:team[:teamname],
+        abbreviation: team[:abbreviation],
+        link: team[:link]
+      }
+    end
+    team_info_hash
```

Comment:
adding a line between the end of the block and the return line helps readability

Reply:
done